lib/index.js

```diff
@@ -36,6 +37,29 @@ class Client {
     throw 'invalid URL argument'
   }
 
+  /**
+   * Make action request CKAN. For more info check
+   * https://docs.ckan.org/en/latest/api/index.html#action-api-reference
+   * @param {string} actionName - The action name, e.g. site_read, package_show ...
+   * @param {object} payload - The payload being sent to CKAN
+   */
+  async action(actionName, payload) {
+    const path = `/api/3/action/${actionName}`
+    const config = {
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+        Authorization: this.authToken,
```

Comment:
To check is `this.authToken` the CKAN API key? I assume it is - in which case we might want to name that more clearly ..